https://issues.redhat.com/browse/ACM-20692 (#7849)
open-cluster-management/rhacm-docs: release_notes/acm_whats_new.adoc

[#governance-whats-new]
== Governance

> * In your policy generator, for the `policies[].manifests[].patches` configuration, you now have a list of Kustomize patches that you can apply to your manifests. 

Reviewer: It's the `openapi` configuration that's new--the `patches` field already existed.

```suggestion
* In your policy generator, for the `policies[].manifests[].patches` configuration, you can now provide an OpenAPI schema for the Kustomize patches applied to your manifests. 
```

> * In your policy generator, for the `policies[].manifests[].patches` configuration, you can now provide an OpenAPI schema for the Kustomize patches applied to your manifests. 

Reviewer 2: This reads fine in my opinion. However, I noticed that you did not link to a source for the user to learn more information. There are also links missing for the 3 entries after this.

> * You can now use the `governance-standalone-hub-templating` add-on to resolve hub templates directly on managed clusters when the `ConfigurationPolicy` is created outside of the policy framework. Previously the policy framework on the hub cluster was required to resolve hub templates. 

Reviewer 2: In the past we did not mention what was done "previously". Is it important here to mention the previous before of the policy framework? I suggest that we remove that line, but not sure what was decided since being on leave.

[#governance-whats-new]
== Governance

> * In your policy generator, for the `policies[].manifests[].patches` configuration, you can now provide an OpenAPI schema for the Kustomize patches applied to your manifests. For more information, see the link:../governance/policy_generator.adoc#policy-generator-yaml-table[Policy Generator configuration reference table].

Reviewer 3: Will the link here work? having ".." doesn't look right to me. It should have the full external URL.

Reviewer 2: this link is correct. i confirmed by using the link check tool and I clicked the link to verify

Reviewer 3: @dockerymick Just for my understanding, will it eventually show as "../" as part of the url to the end-users, and still work?

In other places, I have seen "http://" used instead of "../"

Reviewer: @drkho FYI: The `link:../` are relative links to another local `*.adoc` file in this repo and are resolved when the docs get built.

Reviewer 2: @drkho the link is displayed as hyperlinked text. Here is the preview of the file: https://github.com/stolostron/rhacm-docs/blob/e03f9662e6f347dcf7db88d6cc50d246313e7d54/release_notes/acm_whats_new.adoc#governance-whats-new

Links that are external to the RHACM doc require the full URL. We do not need the full URL in this case because we are pointing to a file within the RHACM doc. @dhaiducek thanks for chiming in, you are correct

Reviewer 3: Cool, didn't realize that the link can be relative. Thanks, @dhaiducek and @dockerymick. I am good. 👍🏼

Reviewer 2: No worries, thanks for asking! Your review is valuable. Here is a link to the ASCII docs for linking: https://docs.asciidoctor.org/asciidoc/latest/macros/link-macro/

> * You can now use the `governance-standalone-hub-templating` add-on to resolve hub templates directly on managed clusters when the `ConfigurationPolicy` is created outside of the policy framework. For more information, see link:../governance/adv_template_process.adoc#resolve-hub-templates[Resolving hub templates after defining configuration policies].

Reviewer 2: Remove the directional language "outside"

```suggestion
* You can now use the `governance-standalone-hub-templating` add-on to resolve hub cluster templates directly on managed clusters when the `ConfigurationPolicy` resource is created externally to the policy framework. For more information, see link:../governance/adv_template_process.adoc#resolve-hub-templates[Resolving hub templates after defining configuration policies].
```
